src/vic3/data/production_methods.rs

```diff
@@ -99,6 +99,19 @@ impl ProductionMethodGroup {
     pub fn add(db: &mut Db, key: Token, block: Block) {
         db.add(Item::ProductionMethodGroup, key, block, Box::new(Self {}));
     }
+
+    #[allow(clippy::unused_self)]
+    pub fn contains_production_method(
+        &self,
+        contains_pm: &Token,
+        block: &Block,
+        _data: &Everything,
+    ) -> bool {
+        if let Some(pms) = block.get_field_list("production_methods") {
+            return pms.iter().any(|pm| pm == contains_pm);
+        }
+        false
```

Comment:
can be replaced by `is_some_and` but not a big deal

Reply:
I think that would actually make the code harder to read, with its nested closures